src/utils/configOverride.ts

```diff
@@ -16,7 +16,10 @@ export function getConfigNavigateCallback(navigate: NavigateFunction) {
     navigate(
       {
         pathname: '/',
-        search: new URLSearchParams([[configQueryKey, configSource]]).toString(),
+        search: new URLSearchParams([
+          [configQueryKey, configSource],
+          ['env', 'prd'],
```

Comment:
Is this needed? I thought the default would be `prd`?

Reply:
Have not finalised it yet. Was testing one of the concepts using preview build. The idea was to preserve `dev` env when navigating within the app. However we still need to reset it when using config selector or config dialog.

Reply:
Can you mark as WIP until its ready to review?